src/hooks/useVotingStats.ts

```diff
@@ -0,0 +1,70 @@
+import useAsyncMemo from 'decentraland-gatsby/dist/hooks/useAsyncMemo'
+
+import { SnapshotGraphql, getQueryTimestamp } from '../clients/SnapshotGraphql'
+import { SnapshotProposal, SnapshotVote } from '../clients/SnapshotGraphqlTypes'
+import { calculateMatch, getChecksumAddress, outcomeMatch } from '../entities/Snapshot/utils'
+import { getPercentage } from '../helpers'
+
+function sortAddressesVotes(votes: SnapshotVote[], userAddress: string | null) {
+  const addressVotes: SnapshotVote[] = []
+  const userVotes: SnapshotVote[] = []
+  votes.forEach((vote) => {
+    if (userAddress && getChecksumAddress(vote.voter!) === getChecksumAddress(userAddress!)) {
+      userVotes.push(vote)
+    } else {
+      addressVotes.push(vote)
+    }
+  })
+  return { addressVotes, userVotes }
+}
+
+function getOutcomeMatchPercentage(addressVotes: SnapshotVote[]) {
```

Comment:
I'd remove this function and do the addresses check on the hook return directly.